src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -50,15 +61,20 @@
  *
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
- * @since 0.15.2
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle MultipleStringLiteralsCheck (500 lines)
  * @todo #306:30min At the moment we don't support HTTP
  *  persistent connections. Would be great to implement
  *  this feature. BkBasic.accept should handle more
  *  than one HTTP request in one connection.
+ * @since 0.15.2
```

Comment:
@dmzaytsev I don't see any repeatable correlation of this statement position. For example in BkParallelTest @since placed last, after @checkstyle annotation
```java
/**
 * Test case for {@link BkParallel}.
 *
 * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
 * @version $Id: da765ae26ed32c19b2361ee48ab723140a1cfa93 $
 * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
 * @since 0.15.2
 */
```

But in FtCLITest placing is changed:
```java
/**
 * Test case for {@link FtCLI}.
 * @author Yegor Bugayenko (yegor@teamed.io)
 * @version $Id: 6c8a3e746b7f0ba3b9010afac6fd3c3403a34036 $
 * @since 0.1
 * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
 */
```

So is there any standartized check for this?